contrib/init/systemd/docker.service

```diff
@@ -5,8 +5,9 @@ After=network.target docker.socket
 Requires=docker.socket
 
 [Service]
-ExecStart=/usr/bin/docker -d -H fd://
+ExecStart=/usr/bin/docker daemon -H fd://
 MountFlags=slave
+Restart=on-failure
```

Comment:
@rhatdan Dan, this was added as part of your implementation (PR 9255). Should we remove this one ?
It was not part of the upstream.